plugins/BEdita/Core/tests/TestCase/Model/Table/ProfilesTableTest.php

```diff
@@ -249,5 +249,6 @@ public function testBeforeSave()
         $success = $this->Profiles->save($profile);
         static::assertTrue((bool)$success);
         static::assertNull($success->get('email'));
+        static::assertEquals('Gustavo Supporto', $success->get('title'));
```

Comment:
Maybe I would add another test with `title` field populated.

Reply:
ok, I'll do it

Reply:
done